### ISSUE TITLE
Contrast Loss did not go down

### ISSUE THREAD

**Issue by EricLina** [LiyaoTang/contrastBoundary#34]
When training on S3DIS using PointTransformer, I find that the contrast loss did not go down, but the CrossEntropy Loss go down normaly. 

Can you provide the detailed trainnig log for me?

**Comment by LiyaoTang** [comment]
Hi,

Thanks for your interest and sorry for my late reply.

We have provided the logs along with the pre-trained model [here](https://drive.google.com/drive/folders/1_ppwnrAu6VRqENTPWPt-3KFqCCTtfsFC).

Best,
Liyao
